apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -195,6 +195,18 @@ p.aiTokenText {
   color: $product_affirmative_default;
 }
 
+h2.studentName {
+  margin-bottom: 8px;
+}
+
+.submittedFeedback {
```

Comment:
tiny nit: this style is just for the feedback in the header right? The name makes it sound a bit like it's for all the feedback, so I'm trying to think if there's a name for this class that communicates that more clearly -- maybe `submittedFeedbackIndicators`? 

I know you're going OOO soon and it would be great to merge this before you do, so don't block on this.